packages/remote-replay-launcher/src/upload-assets-to-s3-and-trigger-test-run.ts

```diff
@@ -0,0 +1,155 @@
+import {
+  createWriteStream,
+  createReadStream,
+  statSync,
+  unlinkSync,
+  existsSync,
+} from "fs";
+import { IncomingMessage } from "http";
+import { request as httpsRequest } from "https";
+import { tmpdir } from "os";
+import { join, resolve } from "path";
+import {
+  completeAssetUpload,
+  getApiToken,
+  requestAssetUpload,
+  createClient,
+} from "@alwaysmeticulous/client";
+import { METICULOUS_LOGGER_NAME } from "@alwaysmeticulous/common";
+import archiver from "archiver";
+import log from "loglevel";
+import {
+  UploadAssetsToS3AndTriggerTestRunOptions,
+  ExecuteRemoteTestRunResult,
+} from "./types";
+
+export const uploadAssetsToS3AndTriggerTestRun = async ({
```

Comment:
naming nit: S3 is an purely an implementation detail so no need to expose it via the API name here.

```suggestion
export const uploadAssetsAndTriggerTestRun = async ({
```